components/SelectionHook/SelectionHook.js

```diff
@@ -16,13 +17,20 @@ const Hook = styled(Image)`
   }
 `;
 
-export default function SelectionHook() {
+export default function SelectionHook({
+  onToggleSelection,
+  isSelected,
+  isSelectionMode,
+}) {
   return (
     <Hook
+      $isSelectionMode={isSelectionMode}
```

Comment:
I don't know this $ in this position. What is it doing?

Reply:
without an $ I did receive this error. 

"Warning: React does not recognize the `isSelectionMode` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isselectionmode` instead. If you accidentally passed it from a parent component, remove it from the DOM element."